bootstrap/Context/FailureContext.php

```diff
@@ -413,12 +412,13 @@ public function takeScreenshot($filename, ElementInterface $page, $driver)
         $content = null;
         $filename .= microtime(true);
         // If not selenium driver, extract the html and put in file.
-        if ($this->screenshotMode === self::SCREENSHOT_MODE_HTML || ! ($driver instanceof Selenium2Driver)) {
+        if ($this->screenshotMode === self::SCREENSHOT_MODE_HTML || ! ($driver instanceof DriverInterface)) {
             $filename .= '.html';
             $content = $this->applySiteSpecificFilters($page->getOuterHtml());
         } else {
-            $filename .= '.png';
             $content = $driver->getScreenshot();
+            $size = getimagesizefromstring($content);
```

Comment:
As commented previously, don't believe this is needed as png's have been consistent and easier to predict the behaviour and tools that can integrate with this extension further. If anything, this could be raised as another PR and we can have a chat about its usefulness thr. I do thank you for your contribution as this PR alone makes things easier :)

Reply:
I included this change cause `Behat\Mink\Driver\DriverInterface` doesn't constraint to a specific image format. I agree though we could have another PR.